tests/Unit/NumericalAlgorithms/LinearOperators/Test_PartialDerivatives.cpp

```diff
@@ -500,6 +501,9 @@ SPECTRE_TEST_CASE("Unit.Numerical.LinearOperators.LogicalDerivs",
   constexpr size_t max_points =
       Spectral::maximum_number_of_points<Spectral::Basis::Legendre> / 2;
   for (size_t n0 = min_points; n0 <= max_points; ++n0) {
+    if (n0 > 6 and n0 != max_points) {
+      continue;
```

Comment:
Maybe add a comment `// To keep test time reasonable we don't check all possible values`?